src/samples/audio/detect_test.c

```diff
@@ -48,21 +49,6 @@
 /* default number of samples before detection is activated  */
 #define KEYPHRASE_DEFAULT_PREAMBLE_LENGTH 0
 
-/* 990 ms of data considering frame rate 16KHz
- * 990 ms = amount of data micro speech nn operates on
- */
-#define KWD_NN_KEY_LEN         15840
-
-#define KWD_NN_NUM_OF_CHANNELS 1
-
-/* ~2 seconds of samples
- * acquisition buffer maximum size
- * used as input for detection algorithms
- * it contains the necessary amount of
- * data required by a specific algorithm
- */
-#define KWD_NN_IN_BUFF_SIZE    (2 * KWD_NN_KEY_LEN * KWD_NN_NUM_OF_CHANNELS)
-
```

Comment:
Lines added in a commit and moved in the next doesn't sound that good. Try to have them in the proper place from the get go.